scipy/optimize/nonlin.py

```diff
@@ -1350,7 +1350,7 @@ class KrylovJacobian(Jacobian):
         preconditioners. For example,
 
         >>> jac = BroydenFirst()
-        >>> kjac = KrylovJacobian(inner_M=jac.inverse).
+        >>> kjac = KrylovJacobian(inner_M=jac.inverse)
```

Comment:
and you actually need to add this in order to not trigger ``refguide_check.py``:

    >>> from scipy.optimize.nonlin import BroydenFirst, KrylovJacobian, InverseJacobian